extensions/binding/org.eclipse.smarthome.binding.onewire/ESH-INF/thing/common.xml

```diff
@@ -17,12 +17,13 @@
 			<parameter name="resolution" type="text">
 				<label>Sensor resolution</label>
 				<options>
+					<option value="">default</option>
 					<option value="9">9 bit</option>
 					<option value="10">10 bit</option>
 					<option value="11">11 bit</option>
 					<option value="12">12 bit</option>
 				</options>
-				<default>10</default>
+				<default></default>
```

Comment:
What happens if you select 9-12 on sensors that do not support it? I would guess it results in a refresh exception. Please check if you can add code to ignore this setting if the sensor does not support it.

Reply:
Yes, it results in the refresh exception which can be easily fixed. But currently you cannot read temperature for some sensors, where there is no resolution option in OWFS.

I can analyse if the check can be done and is worth doing it.